### ISSUE TITLE
Move the SAML adapter to jakarta and remove old javax integration

### ISSUE THREAD

**Issue by rmartinc** [keycloak/keycloak#30556]
### Description

Once #30555 is merged and the testsuite uses wildfly 29 the SAML adapter can be fully jakartaee and no old javax/javaee integration is needed. This means that keycloak does not need to maintain two versions (javax and jakarta) for several projects (in the [saml adapters folder](https://github.com/keycloak/keycloak/tree/25.0.0/adapters/saml) `core-jakarta` and `wildfly-elytron-jakarta`), they can now be merged into just one project. The initial idea is removing the transformation project (the `-jakarta` one) and make the original project jakarta compatible (but it can be done the other way around, maintain the `-jakarta` and remove the original folder).
